src/main/java/Ryan.java

```diff
@@ -0,0 +1,58 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Ryan {
+    public static void main(String[] args) {
+        boolean exit = false;
+        Scanner scanner = new Scanner(System.in);
+        ArrayList<Task> tasks = new ArrayList<>();
+
+        Utils.horizontalLine();
+        System.out.println("Hello! I'm Ryan\nWhat can I do for you?");
+        Utils.horizontalLine();
+
+        while (!exit) {
+            String command = scanner.nextLine();
+
+            if (command.equals("bye")) {
```

Comment:
Conditional curly braces are correctly wrapped in curly brackets, which is great. This prevents subtle bugs.